include/blobify/load.hpp

```diff
@@ -13,6 +13,7 @@
 #include <magic_enum.hpp>
 
 #include <cstddef>
+#include <vector>
```

Comment:
What definition from `<vector>` is needed by load.hpp? I can't see any use of `std::vector` here.

If only some of the files that include `load.hpp` require `<vector>`'s definitions, would you mind including the header in those files instead?

Reply:
I just realized that `examples/bmp.cpp` was were `#include <vector>` should actually belong.